money.go

```diff
@@ -91,6 +93,30 @@ func (m Money) Div(n Money) Money {
 	return Money{C: m.C, M: Rnd(i, f-float64(i))}
 }
 
+func (m Money) Split(chunks int64) []Money {
```

Comment:
Can you add a doc comment explaining what the function does, to match the rest of the public functions in here?